.github/workflows/reusable-build.yml

```diff
@@ -38,7 +38,7 @@ permissions:
 
 jobs:
   build:
-    timeout-minutes: 60
+    timeout-minutes: 90
```

Comment:
will this bump the timeout for all builds not just codeql?   Should just codeql be longer?

Reply:
I don't think this can be set based on the parameters of the job, need to investigate.